### ISSUE TITLE
Managesieve not working after update from 1.1.x to 1.2.x

### ISSUE THREAD

**Issue by zimmski** [roundcube/roundcubemail#5570]
I have updated my Roundcube instance form 1.1.x to 1.2.3 and now the managesieve plugin (version 8.7) is not working anymore. When I go to the "Filters" configuration in the UI, I receive a 500 status response. Setting the logging level to the maximum value then reveils the following problem: "Fatal error: Call to undefined method rcube_utils::parse_socket_options() in .../plugins/managesieve/lib/Roundcube/rcube_sieve_engine.php on line 199"

I already searched for the problem but could not find anything.

**Comment by alecpl** [comment]
Managesieve 8.7 is not the one included in Roundcube 1.2.3. The version you're using is not compatible with 1.2.x.

**Comment by zimmski** [comment]
I always use Roundcube's updater so I am confused why this happend. Anyway, the mangesieve plugin still does not work. So I like to reopen this issue.

I manually replaced the plugins directory with the one of the 1.2.3 release tar. The plugin is now at "8.6". And I now get the error "Fatal error: Class 'Net_Sieve' not found in .../plugins/managesieve/lib/Roundcube/rcube_sieve.php on line 65".

I checked using "pear" If I meet the requirement of the INSTALL file ("Net_Sieve 1.3.2 or newer (for managesieve plugin)"). It is at 1.3.4.
